javascript/javascript2/week1/movieApp/features.js

```diff
@@ -154,10 +184,43 @@ function displayMovies(movies) {
     heroSection.appendChild(noMoviesFoundPara);
   }
 }
+function moviesByYear() {
+  const sortedMoviesbyYear = movieList.sort(
```

Comment:
Be careful, `sort` does not return a new array. It changes `movieList` directly.